tests/mbedtls_test/main.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (C) 2021_ML!PA Consulting GmbH
+ */
+/**
+ * @ingroup     test app lwip_mbedtls
+ * @{
+ *
+ * @file
+ * @brief       send encrypted data to the server
+ *
+ * @}
+ * @ author Mariem Charrada <mariem.charrada@ml-pa.com>
+ */
+
+#include "net/sock/tcp.h"
+#include "shell.h"
+#include "msg.h"
+#include "xtimer.h"
+
+#include "lwip/debug.h"
+#include "mbedtls/debug.h"
+#include "net_sockets.h"
+#include "ssl.h"
+#include "entropy.h"
+#include "ctr_drbg.h"
+#include "x509_crt.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+#define MAIN_QUEUE_SIZE (4)
+
+static void _lwip_mbedtls_my_debug( void *ctx, int level, const char *file, int line, const char *str )
+{
+	((void) level);
+	fprintf( (FILE *) ctx, "%s:%04d: %s", file, line, str );
+	fflush( (FILE *) ctx );
+}
+
+static const unsigned char ca_certificate[2048]=
```

Comment:
The string is less than 2048 characters long - no need to specify the size of the array here. 

```suggestion
static const unsigned char ca_certificate[]=
```